### PR TITLE
added default -doMajorMinor 1 to output angsd beagle format 

### PR DESCRIPTION
**Pull request by alexandregilardet**
+ corrected typo in error message about angsd output format

To output ANGSD beagle format, the argument -doMajorMinor to infer major and minor alleles is required. For now, eager would output a corrupted .glf.gz file if 'beagle' is selected as output with the following .command.err:
-> For dumping beaglestyle output you need to estimate major/minor: -doMajorMinor
I added as default the -doMajorMinor 1 argument which I'm more familiar with (http://www.popgen.dk/angsd/index.php/Major_Minor).

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test_tsv,crick`).

### CODE REVIEW

**Comment by apeltzer** [comment]
You will need to add this change to the `CHANGELOG.md` too to make the tests pass 👍🏻

**Comment by alexandregilardet** [comment]
Ok I THINK I got it right now...

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,18 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
-- [#626](https://github.com/nf-core/eager/issues/626) - Add additional checks to ensure pipeline will give useful error if cells of a TSV column are empty
-- Added note to documentation that when using `--skip_collapse` this will use _paired-end_ alignment mode with mappers when using PE data.  `
-- [#673](https://github.com/nf-core/eager/pull/673) Fix Kraken database loading when loading from directory instead of compressed file.
-- [#688](https://github.com/nf-core/eager/issues/668) - Allow pipeline to complete, even if Qualimap crashes due to an empty or corrupt BAM file for one sample/library
-- [#683](https://github.com/nf-core/eager/pull/683) - Sets `--igenomes_ignore` to true by default, as rarely used by users currently and makes resolving configs less complex.
-- Added exit code `140` to re-tryable exit code list to account for certain scheduler wall-time limit fails.
-- [672](https://github.com/nf-core/eager/issues/672) - Removed java parameter from picard tools which could cause memory issues
-- [679](https://github.com/nf-core/eager/issues/679) - Refactor within-process bash conditions to groovy/nextflow, due to incompatibility with some servers environments
```
Comment (jfy133) [inline CHANGELOG.md:23]: Was this all meant to be deleted? Your change should just be appened to the endof the list.

**Review by jfy133 — CHANGES_REQUESTED** [review]
Good catch on the typos in the validation section.

There was a mistake in the changelog I think, but otherwise I think we are good to go.

**Review by jfy133 — APPROVED** [review]

**Comment by jfy133** [comment]
Looks good to me! Merging!

**Comment by jfy133** [comment]
Thanks very much @alexandregilardet ! Feel free to make an extra PR adding yourself to the `contributors` list on the main README if you're happy to do so :)

**Pull request merged**